src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -44,9 +47,18 @@ public AddCommand parse(String args) throws ParseException {
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
 
-        Person person = new Person(name, phone, email, address, tagList);
+        // TODO: do add functionality of nok
+        Nok placeholderNok =
+                new Nok(
+                    new Name("Bernice Yu"),
+                    new Phone("99272758"),
+                    new Email("berniceyu@example.com"),
+                    new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18")
+                );
 
-        return new AddCommand(person);
+        Student student = new Student(name, phone, email, address, placeholderNok, tagList);
+
+        return new AddCommand(student);
```

Comment:
So every student added will have bernice as a deafult NOK contact?

Reply:
Ah, this is just a placeholder so that the test cases pass. I've only done the edit command. Could you do the add command? Thank you!